stars/game.py

```diff
@@ -108,7 +113,10 @@ def __init__(self, x=50, y=50, z=50, num_systems=10, system_names=None, races=[]
                 for p in self.players:
                     p.add_intel(b, {'location': b.location, 'size': b.radius})
             self._scan([])
-            self._call(self.players, 'update_stats')
+            if self.hundreth == 0:
+                self.calculate_score()
```

Comment:
If we only calculate score once per generation, are we accepting that it might miss conditions where a player would win in one hundreth but by the end they are no longer in a winning condition?